docker-build

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
Please use `-e`

Reply:
OK; to be clear, house style is:

```
#!/bin/bash -e
```

instead of

```
#!/bin/bash
set -e
```
? Happy to do whichever.

Reply:
I'm ok with either. `set -e` is a little better since you can add a comment at the same time that spells out that errors are expected to terminate immediately.